src/script_lua.c

```diff
@@ -1504,9 +1505,16 @@ void luaRegisterRedisAPI(lua_State* lua) {
     lua_pushcfunction(lua,luaRedisAclCheckCmdPermissionsCommand);
     lua_settable(lua,-3);
 
-    /* Finally set the table as 'redis' global var. */
+    /* Finally set the table as 'redis' global var. 
+     * We will also alias it to 'server' global var to remove redis from code base. 
+     * In future major version release, we can remove 'redis' global var entirely. */
     lua_setglobal(lua,REDIS_API_NAME);
-
+    // lua_getglobal invocation retrieves the 'redis' variable value to the stack. 
+    // lua_setglobal invocation uses the value from stack to set 'server' global variable. 
+    // This is not a deep copy but is enough for our purpose here. 
```

Comment:
```suggestion
    /* lua_getglobal invocation retrieves the 'redis' variable value to the stack. 
     * lua_setglobal invocation uses the value from stack to set 'server' global variable. 
     * This is not a deep copy but is enough for our purpose here. */
```

There is a convention of always using multi-line comments. (Now we're independent, we can actually add a rule to enforce it.

Reply:
Ok tried to merge your code change but it created a new commit, so instead pushing a new merged commit.